docs/Readme.md

```diff
@@ -1,119 +1,216 @@
-# App-AutoScaler User Guide
 
-The `App-AutoScaler` provides the capability to automatically adjust the instance number of Cloud Foundry applications through
+# Scaling an Application Using App AutoScaler
+This topic describes how to configure and use [Cloud Foundry App Auto-Scaler][git] to scale your application **horizontally** and automatically based on the scaling policy that you set.
 
-* Dynamic scaling based on application performance metrics
-* Scheduled scaling based on time
+---
+## Overview
 
-----
-## Terminology
+The [Cloud Foundry App Auto-Scaler][git] provides the capacity to : 
+* Adjust the application's instance number dynamically based on application metrics' thresholds, such as memory usage, throughput, response time and etc.
```

Comment:
Adjust the application's instance number dynamically based on performance metrics of the application.